nsls2forge_utils/dashboard.py

```diff
@@ -74,25 +68,48 @@ def create_dashboard(names=None, write_to='README.md'):
     row_string = ('|[{name}](https://github.com/nsls-ii-forge/{name}-feedstock)|{build} <br/> {health}'
                   '|{nsls_version} <br/> {pypi_version} <br/> {defaults_version} <br/> '
                   '{cf_version} <br/> {github_version}|{downloads}|\n')
-    description = '''# Project Management\nReleases, Installers, Specs, and more!\n'''
     header = ('# Feedstock Packages Build Status\n\n'
               '| Repo | Build <br/> Health | nsls2forge <br/> PyPI <br/> defaults <br/> conda-forge <br/>'
               ' GitHub <br/> Versions | Downloads|\n|:-------:|'
               ':-----------:|:---------------:|:--------------:|\n')
 
-    out = description
-    if names is None:
-        pkgs = sorted(get_all_feedstocks(organization='nsls-ii-forge'))
-    else:
-        pkgs = sorted(get_all_feedstocks(cached=True, filepath=names))
-    out += header
-    for pkg in pkgs:
+    dashboard = header
+    for pkg in names:
         print(f'Formatting {pkg}...')
         org, repo = _extract_github_org_and_repo(pkg)
         if repo == '':
             repo = pkg
         tmp = row_string.format(**main_format, name=pkg, org=org, repo=repo)
-        out += tmp.format(name=pkg, org=org, repo=repo)
+        dashboard += tmp.format(name=pkg, org=org, repo=repo)
+    return dashboard
+
+
+def create_dashboard(names=None, write_to='README.md'):
+    '''
+    Creates a table of packages with their build status, health, conda-forge version,
```

Comment:
Should the PyPI, GitHub and Anaconda's defaults be mentioned here as well?

Reply:
Yes, forgot to update this. Making the change now.